### DISCUSSION TITLE
Sandbox import error: ModuleNotFoundError: No module named 'requests' error: exit status 255

### DISCUSSION THREAD

**Discussion by EvanGerry** [langgenius/dify#4854] [General]
For path has requests dependency, why can't import, sandbox image: langgenius/dify-sandbox:latest
![image](https://github.com/langgenius/dify/assets/14178306/da99544c-30ff-4173-b4f0-3c2f4d86d3e4)

![image](https://github.com/langgenius/dify/assets/14178306/ee63475e-7be7-4dc2-a0cd-b3f8add5c857)

**Comment by patryk20120** [comment]
You can use only preloaded packages. They are listed here: https://github.com/langgenius/dify/pull/3567/files

**Reply by EvanGerry** [reply]
thanks your reply, i will try Http Request component

**Comment by rosehuynh92** [comment]
> Đối với đường dẫn có yêu cầu phụ thuộc, tại sao không thể nhập, hình ảnh hộp cát: langgenius/dify-sandbox:latest ![hình ảnh](https://private-user-images.githubusercontent.com/14178306/335790797-da99544c-30ff-4173-b4f0-3c2f4d86d3e4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNzQyOTcsIm5iZiI6MTcyNzE3Mzk5NywicGF0aCI6Ii8xNDE3ODMwNi8zMzU3OTA3OTctZGE5OTU0NGMtMzBmZi00MTczLWI0ZjAtM2MyZjRkODZkM2U0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDEwMzMxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0ZjA1ZjZkY2I5YTZmOWRhYTFkMjRmNjFjNDAxMzRiYmQ0NjgzNGUzNDhiYjA3YmU3NzAyYjViNWEyYTA4MmUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.2hTw0ulAGT0jVg42kWudpbw0OyiqiAOuuyFPiSmFx2U)
> 
> ![hình ảnh](https://private-user-images.githubusercontent.com/14178306/335790741-ee63475e-7be7-4dc2-a0cd-b3f8add5c857.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjcxNzQyOTcsIm5iZiI6MTcyNzE3Mzk5NywicGF0aCI6Ii8xNDE3ODMwNi8zMzU3OTA3NDEtZWU2MzQ3NWUtN2JlNy00ZGMyLWEwY2QtYjNmOGFkZDVjODU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTI0VDEwMzMxN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMyYzZmNzE1YjQyZDUzYTEzYmMwYjcwMGUzZTkxZDc0MmIxZWFkMzNlNmZlNjcxNTk0ZjI3OWE2MTIzOGExMTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.O1L0BcZzwYNCUomsXRCY21zR0h8PTWCObPH2-byLN20)
